src/bindings/python/tests/test_ngraph/test_rdft.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import openvino.runtime.opset9 as ov
+from openvino.runtime import Shape
+import numpy as np
+from tests.runtime import get_runtime
+
+np.random.seed(0)
+
+def test_rdft_1d():
+    runtime = get_runtime()
+    n = 50
+    shape = [n]
+    data = np.random.uniform(0, 1, shape).astype(np.float32)
+    param = ov.parameter(Shape(shape), name="input", dtype=np.float32)
+    input_axes = ov.constant(np.array([0], dtype=np.int64))
+
+    node = ov.rdft(param, input_axes)
```

Comment:
I see tests for 2D-rdft with `signal_size` param, is it by design that 1D case is missing?

Reply:
It's by design. Signal size test cases are just to make sure python bindings respect that parameter in the API.